### PR TITLE
Recompiled xtend - revert whitespace changes in generated code

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: org.eclipse.xtext.xbase.lib.tests/xtend-gen/org/eclipse/xtext/xbase/tests/lib/ArrayExtensionsCharTest.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License 2.0 which is available at
  * http://www.eclipse.org/legal/epl-2.0.
- *
+ *
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.lib.tests/xtend-gen/org/eclipse/xtext/xbase/tests/lib/ArrayExtensionsCharTest.java]: hmm that space is not there in the original template. does this come from xtend or from the comment in the xtend file itself

Comment (cdietrich) [reply org.eclipse.xtext.xbase.lib.tests/xtend-gen/org/eclipse/xtext/xbase/tests/lib/ArrayExtensionsCharTest.java]: https://www.eclipse.org/projects/handbook/#ip-copyright-headers

**Comment by miklossy** [comment]
I would prefer not to include that whitespace at all.

**Comment by cdietrich** [comment]
well there is a newline in the template but no extra space.
thus i wonder if this needs to be fixed in
the original files or if it is added by the generator?

**Comment by szarnekow** [comment]
> the original files or if it is added by the generator?

It's the Xtend compiler. EMF codegen suffers from the same issue, though.

**Pull request merged**